packages/scripts/src/application.ts

```diff
@@ -549,6 +549,7 @@ export class Application {
         configName?: string;
         entryPoints: Record<string, Record<string, string>>;
     }) {
+        const outputDirInBasePath = this.outputPath.includes(this.basePath);
```

Comment:
if anything, startsWith.

Reply:
I'm writing "if anything" cuz this isn't safe on windows due to case insensitive casing.